federatedscope/core/configs/cfg_fl_algo.py

```diff
@@ -25,6 +25,14 @@ def extend_fl_algo_cfg(cfg):
     cfg.fedprox.use = False
     cfg.fedprox.mu = 0.
 
+    # ---------------------------------------------------------------------- #
```

Comment:
I am wondering whether it is appropriate to put SWA here. It is not an FL algorithm but just a trick to produce another solution from those in the late stage of a training course.